eulers/e20/javascript/AsteriHoku/FactorialDigitSum.js

```diff
@@ -0,0 +1,14 @@
+const bigNum = 100n;//TODO define your number here 
+
+function getFactorial(n) {
```

Comment:
This is good code. One tiny thing to be cautious of...

You don't guarantee/ensure that `n` is a BigInt() when you call this the first time. In strict mode, this will throw a TypeError and tell you that you can't mix types (n to BigInt).

One thing that I like to do in untyped languages is to verify type in the first line of code.
```suggestion
function getFactorial(n) {
	n = BigInt(n);
```
Something like that is almost always super safe and doesn't have a huge performance hit.